manifests/Speedcrunch/Speedcrunch/0.12.yaml

```diff
@@ -1,14 +1,14 @@
-Id: SpeedCrunch.SpeedCrunch
-Version: 0.12
-Name: SpeedCrunch
-Publisher: SpeedCrunch
-License: GPL-2.0-or-later
-LicenseUrl: https://bitbucket.org/heldercorreia/speedcrunch/src/master/LICENSE
-AppMoniker: speedcrunch
-Description: A high-precision scientific calculator featuring a fast, keyboard-driven user interface.
-Homepage: https://speedcrunch.org/
-Installers:
-  - Arch: x86
-    Url: https://bitbucket.org/heldercorreia/speedcrunch/downloads/SpeedCrunch-0.12-win32.exe
-    Sha256: 702C362C9A02C3A79D018A4D68428DC80FAD311ADAD506CC47B69927E47DBF89
-    InstallerType: nullsoft
+Id: SpeedCrunch.SpeedCrunch
```

Comment:
was this intentional?

Reply:
Not really, train was pulling up to my station when I pushed the branch. Can't merge this PR until a way to install via PS1 or nupkg is implemented anyway. Will need to back this change out before merging.